alembic/versions/4ac06dd31afa_backfill_is_correct_from_status.py

```
@@ -0,0 +1,59 @@
+            FROM pick
+            WHERE is_correct IS NULL
+              AND status IN ('correct', 'incorrect')
+            """
+        )
+    ).scalar()
+
+    if result > 0:
+        raise RuntimeError(
+            f"Migration failed: {result} rows with status 'correct' or "
+            "'incorrect' have NULL is_correct."
+        )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)